### PR TITLE
87201 login redirect

### PR DESCRIPTION
**Pull request by numbap**
## [ADO-87201](https://dev.azure.com/VP-BD/DECD/_workitems/edit/87201)

Fixed [AB#87201](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/87201)

### Description

List of proposed changes:

- Got rid of white login screen
-

### CODE REVIEW

File: pages/api/auth/[...nextauth].js [hunk 1]
```diff
@@ -70,4 +70,7 @@ export default NextAuth({
       logger.debug(metadata)
     },
   },
+  pages: {
+    signIn: '/signedout',
+  },
```
Comment (DaceyTom2) [inline pages/api/auth/[...nextauth].js:75]: What does this do?

File: pages/signedout.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { useEffect } from 'react'
```
Comment (DaceyTom2) [inline pages/signedout.js:1]: Can you double check this page meets accessibility?

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]

File: __tests__/components/Layout.test.js [hunk 3]
```diff
@@ -10,6 +10,18 @@ jest.mock('next/router', () => ({
...
+jest.mock('next-auth/react', () => {
+  return {
+    signOut: jest.fn(),
+  }
+})
+
+// // mocks signOut
+// jest.mock('next-auth/react', () => ({
+//   signOut: jest.fn(),
+// }))
```
[the remark this answers is not in the document]
Comment (numbap) [reply __tests__/components/Layout.test.js:23]: fixed

File: pages/_app.js [hunk 4]
```diff
@@ -9,6 +9,9 @@ export default function MyApp({ Component, pageProps }) {
   if (Component.getLayout) {
     return Component.getLayout(<Component {...pageProps} />)
   }
+
+  console.log(Component)
```
[the remark this answers is not in the document]
Comment (numbap) [reply pages/_app.js:13]: Done.

File: pages/signedout.js (hunk 2 above)
Comment (numbap) [reply pages/signedout.js:1]: Tested with screen reader and added H1

File: pages/api/auth/[...nextauth].js (hunk 1 above)
Comment (numbap) [reply pages/api/auth/[...nextauth].js:75]: This is what makes next-auth redirect to the custom signIn page.

**Review by numbap — COMMENTED** [review]
Done.

File: __tests__/components/Layout.test.js [hunk 5]
```diff
@@ -10,6 +10,18 @@ jest.mock('next/router', () => ({
...
+  return {
+    signOut: jest.fn(),
+  }
+})
+
+// // mocks signOut
+jest.mock('next-auth/react', () => ({
+  signOut: jest.fn(),
+}))
+
```
Comment (DaceyTom2) [inline __tests__/components/Layout.test.js:24]: I believe this code wasn't needed at all. We only need to mock signout once

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]
Please remove the minikube-linux-amd64. If you'd like to keep it locally you can add it to git ignore

**Review by DaceyTom2 — APPROVED** [review]

**Pull request merged**